dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -8,12 +8,21 @@ namespaces:
     metrics:
       response:
         type: timer
-        tags: [channel-name, service-name]
-        docs: Request time, note that this does not include time spent reading the response body.
-      response.error:
-        type: meter
-        tags: [channel-name, service-name, reason]
-        docs: Rate of errors received by reason and service-name. Currently only errors with reason `IOException` are reported.
+        tags: [channel-name, service-name, endpoint, status]
+        docs: |
+          Request time split by status. These metrics and their definitions are mostly useful to dialogue developers,
+          to measure the success of routing and retry strategies. Possible status values are:
+          * success: always excludes time spent reading the response body. This classifies requests,
+            which we successfully routed, maybe retried and returned to the user.
+            This does not mean that the ultimate user might not see an exception, e.g. 404, 401 will still potentially
```

Comment:
I think we should filter out non-2xx responses from this case to avoid poisoning the data -- if needs be we can add a `success-with-errors` type tag later on. What do you think?

Reply:
Alright so we want literally:

1. success -> 2xx
2. failure -> QoS, 500 if retryable, IO Exception

Everything else filter?

Reply:
Done